R/data.R

```diff
@@ -1,12 +1,14 @@
 #' Demographic Dataset-updated
 #'
 #' An updated SDTM DM dataset with pediatric patients
-#'
+#' @keywords datasets
```

Comment:
@rossfarrugia mimicing admiral way here as well

Reply:
@zdz2101 i guess we need to add this to man/roxygen/meta.R as well?